### PR TITLE
Update dsp dependencies

### PR DESCRIPTION
**Pull request by kilchenmann**
Added latest version of DSP-JS ([v1.0.0-rc.14](https://www.npmjs.com/package/@dasch-swiss/dsp-js/v/1.0.0-rc.14)) and DSP-UI lib ([v1.0.0-rc.10](https://www.npmjs.com/package/@dasch-swiss/dsp-ui/v/1.0.0-rc.10))

Additional I updated the code because of breaking styling change in ui-lib (search panel) and new feature in js-lib:

- Implementation of redesigned search-panel needed some adjustments in styling and navigation:
  - route `search/advanced` and route `search/expert` are not needed anymore. 
  - corresponding components `advanced-search` and `expert-search` are not needed anymore. It was anyway redundant code

- Implementation of new `healthEndpoint` from DSP-JS to get DSP-API version

### CODE REVIEW

File: src/app/main/header/header.component.ts [hunk 1]
```diff
@@ -100,6 +100,8 @@ export class HeaderComponent implements OnInit, OnDestroy{
     }
 
     doSearch(search: SearchParams) {
+
+        console.log(search);
```
Comment (mdelez) [inline src/app/main/header/header.component.ts:104]: can this be removed?

**Review by mdelez — APPROVED** [review]
LGTM! Thanks :)

File: src/app/main/header/header.component.ts (hunk 1 above)
Comment (kilchenmann) [reply src/app/main/header/header.component.ts:104]: Ups, yes. I will remove it now and merge it asap. So, I can create new release afterwards.

Thanks @mdelez for reviewing it

Comment (kilchenmann) [reply src/app/main/header/header.component.ts]: Done in faa233d

Comment (kilchenmann) [reply src/app/main/header/header.component.ts]: Damn...the tests failed. But why?

**Comment by kilchenmann** [comment]
@mdelez I had to fix some tests especially missing imports in some spec files. Would you like to review again? Thanks

**Review by mdelez — APPROVED** [review]

**Pull request merged**